src/main/scala/pirate/Runners.scala

```diff
@@ -16,10 +16,13 @@ trait Runners {
         }
         case ParseErrorShowHelpText(s) => IO {
           s match {
-            case None      => Console.err.print(Usage.print(command, ctx))
-            case Some(sub) => Console.err.print(Usage.print(command, sub :: ctx))
+            case None      => Console.out.print(Usage.print(command, ctx))
+            case Some(sub) => Console.out.print(Usage.print(command, sub :: ctx))
```

Comment:
I haven't seen any error case which would need to be printed as an error but again I think that points toward a problem in how we are implementing help and version at the moment.